app/models/empire.js

```diff
@@ -79,10 +82,10 @@ export default Model.extend({
   }),
 
   // Cannot do mapBy/sum (yet) because capital is three-building-in-one
```

Comment:
Also, because you want to exclude the capital from the count. Might as well change the comment